data/model/repo_mirror.py

```diff
@@ -114,10 +114,13 @@ def release_mirror(mirror, sync_status):
     sync will be moved ahead as if it were a success. This is to allow a daily sync, for example, to
     retry the next day. Without this, users would need to manually run syncs to clear failure state.
     """
+    retries = None
```

Comment:
Default this to 0 instead. Is it's `None` and the status is `FAIL`, the call to `max()` will throw an exception.

Reply:
Yes, makes sense. I'll change the default initialization.

Reply:
So, I am initializing 'retries' to `float('-inf')` so one can use the variable to differentiate between `RepoMirrorStatus.FAIL` and `RepoMirrorStatus.SUCCESS` and we do not run into exception on `max()`
Say, the `sync_status` passed to the method is `RepoMirrorStatus.FAIL` and defaulting `retries` to 0 allows the if block at https://github.com/quay/quay/blob/master/data/model/repo_mirror.py#L120 to execute which is not the behavior we want.